### PR TITLE
Errormsg styling and css fixes

### PR DESCRIPTION
**Pull request by NeelliJ**
Realized this is a bit large for a PR. Main thing though, error messages. Two very different types.
- Error pages (404, something else went wrong), commits 5-6
- Diy Form messages (success, error), commits 3 & 7 (=last one)

The Form error message solution might change later. Not sure of best solution (see comments)

### CODE REVIEW

**Comment by NeelliJ** [comment]
Ok ahem. The two additional commits = changes form error message placing to above the form. And scrolls to the h1 just above it, to make users aware of the message

File: myko/src/lib/components/DiyView.svelte [hunk 1]
```diff
@@ -50,14 +59,30 @@
...
       </div>
 
       <div>
-        <input type="submit" value="Notera aktiviteten" required class="btn secondary" />
+        <input
+          type="submit"
+          value="Notera aktiviteten"
+          required
+          class="btn"
+          action="h1"
```
Comment (zamzterz) [inline myko/src/lib/components/DiyView.svelte:72]: There is a "Svelte-way" to doing this, avoiding hardcoding the element and only scrolling to the error message element if it is shown (so not when form is successfully posted). I tested it [here](https://github.com/Relationsinstitutet/myko/pull/99) and seems to work nicely.

**Review by zamzterz — COMMENTED** [review]
Looks great (and the `confused-cat` ❤️ )!

Just some small questions and a suggestion for how to scroll the DiyView error message into view in a "Svelte-way". 🙂

File: myko/src/lib/components/FullPageModal.svelte [hunk 2]
```diff
@@ -62,6 +57,11 @@
     height: 100%;
   }
 
+  .done-btn,
+  .help-text {
+    z-index: 1;
```
[the remark this answers is not in the document]
Comment (NeelliJ) [reply myko/src/lib/components/FullPageModal.svelte:62]: my bad, missed removing this!

File: myko/src/lib/components/DiyView.svelte (hunk 1 above)
Comment (NeelliJ) [reply myko/src/lib/components/DiyView.svelte]: Great that's so cool! Will have to read up on those bindings

Comment (NeelliJ) [reply myko/src/lib/components/DiyView.svelte]: Hm realized now on testing that the error message doesn't go away. If you would first not fill the form in correctly, but then upon correction do it right, you end up with both error and success messages.  
Not sure how to fix this

Comment (zamzterz) [reply myko/src/lib/components/DiyView.svelte]: Nice catch! I made another PR fixing that: https://github.com/Relationsinstitutet/myko/pull/101

**Pull request merged**